src/main/java/bio/terra/service/snapshotbuilder/utils/AggregateSynapseQueryResultsUtils.java

```diff
@@ -38,4 +38,14 @@ public static int toCount(ResultSet rs) {
           "Error processing result set into SnapshotBuilderConcept model", e);
     }
   }
+
+  public static String toDomainId(ResultSet rs) {
+    try {
+      // Azure ResultSet is 1 indexed
+      // https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html
+      return rs.getString(1);
```

Comment:
Nit: I believe if you are getting a specific column, you can use the column name here - the count couldn't do that because it was a generated column, so the name wasn't consistent/understandable (although I think we added aliasing so that should be okay too)
https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html#getString(java.lang.String)

Reply:
I wouldn't worry about changing `count` in this PR - we can just file a tech debt ticket for it.

Reply:
I filed: https://broadworkbench.atlassian.net/browse/DC-971

Reply:
I had really hoped I had unit tested this, looks like I have not. I can add adding it into the integration tests and connected tests as part of that ticket.